assets/lambda/new_s3obj_queue_notifier_lambda.py

```diff
@@ -144,39 +56,14 @@ def index_handler(event, context):
     """
 
     queue_name = os.getenv("QUEUE_NAME")
-    etl_attrs_ddb_name = os.getenv("ETL_ATTRS_DDB_TABLE")
 
     # obligatory data validation
     if queue_name is None:
         msg = "No queue name provided. Cannot insert notification message."
-        logger.error(msg)
         return {"statusCode": 500, "body": msg}
 
-    if etl_attrs_ddb_name is None:
-        msg = (
-            "No ETL attributes DynamoDB table name provided. Cannot insert "
-            "notification message."
-        )
-        logger.error(msg)
-        return {"statusCode": 500, "body": msg}
-
-    # grab all the info we care about for the new s3 object from the event
-    # object.
-    # NOTE: the event structure for notifications can contain many records.
-    #       grabbing them all to handle potential of multiple uploads in a
-    #       batch. a new message will be inserted for each record
-    object_info = [
-        {
-            "bucket": rec["s3"]["bucket"]["name"],
-            "key": urllib.parse.unquote_plus(
-                rec["s3"]["object"]["key"], encoding="utf-8"
-            ),
-        }
-        for rec in event["Records"]
-    ]
-
     # get a reference to the etl attributes table
-    ddb_table = get_etl_attrs_table(etl_attrs_ddb_name)
+    ddb_table = EtlTable()
 
     try:
```

Comment:
I don't follow this at all, will need clarification

Reply:
oh wait I see what you mean, I didn't get get rid of it. It was never used in the other one 🤷🏼 I'm not sure since I didn't originally write that code.

Reply:
i'll chalk it up to an overzealous reuse of code by the original author. whomever that joker is :sweat_smile: